apps/docs/content/components/forms/NumberField.mdx

```diff
@@ -0,0 +1,95 @@
+---
+title: NumberField
+description: A number field is a specialized number field that allows a user to enter a number.
```

Comment:
this sound weird. maybe it should say "a specialized text field".

Reply:
or even input field

Reply:
yep you are right